Pod/Classes/TMPodspec.m

```diff
@@ -61,21 +76,125 @@ - (NSString *)version {
     return [self stringForKey:PodspecKeyVersion];
 }
 
-- (NSString *)summary {
-    return [self stringForKey:PodspecKeySummary];
+- (NSArray *)authors {
+    NSString *authorName = [self stringForKey:PodspecKeyAuthor];
+    NSArray *authorNames = [self arrayForKey:PodspecKeyAuthors];
+    NSDictionary *authorNamesAndAddresses = [self dictionaryForKey:PodspecKeyAuthors];
+    
+    NSMutableArray *authors = [NSMutableArray array];
+    
+    if (authorNamesAndAddresses) {
+        [authorNamesAndAddresses enumerateKeysAndObjectsUsingBlock:^(id name, id emailAddress, BOOL *stop) {
+            if ([name isKindOfClass:[NSString class]] && [emailAddress isKindOfClass:[NSString class]]) {
+                TMAuthor *author = [[TMAuthor alloc] initWithName:(NSString *)name emailAddress:(NSString *)emailAddress];
+                [authors addObject:author];
+            }
+        }];
+    }
+    else if (authorNames) {
+        for (id name in authorNames) {
+            if ([name isKindOfClass:[NSString class]]) {
+                TMAuthor *author = [[TMAuthor alloc] initWithName:(NSString *)name emailAddress:nil];
+                [authors addObject:(NSString *)author];
+            }
+        }
+    }
+    else if (authorName) {
+        TMAuthor *author = [[TMAuthor alloc] initWithName:authorName emailAddress:nil];
+        [authors addObject:(NSString *)author];
+    }
+    else {
+        return nil;
```

Comment:
Don't feel *too* strongly but why not just return the empty array in this case? Similar to what `NSFetchRequest` does.